src/Main.hs

```diff
@@ -39,5 +39,5 @@ main = do Options f c conf <- execParser opts
           cpg <- flip runReaderT cfg $ do
             cs       <- contracts f
             (v,w,ts) <- loadSpecified (pack . (f ++) . (':' :) <$> c) cs >>= prepareForTest
-            ui v w ts (Just $ mkGenDict 0.15 (extractConstants cs) [] g)
+            ui v w ts (Just $ mkGenDict 0.15 (extractConstants cs) [] g (returnTypes cs))
```

Comment:
At some point we need to start adding everything extracted from the contracts into some more suitable type..

Reply:
Yes definitely! I think that escapes the scope of this PR though